README.md

```diff
@@ -4,7 +4,27 @@ TypeScript based types used in PNC project.
 
 This project clones **PNC project** and builds `pnc/rest-api` module to get `pnc-openapi.json` specification using `generate-openapi-spec.sh` script and transform it to `index.d.ts` using `dtsgen` command.
 
-## Generate new types and release new version
+## Releasing new version of pnc-api-types-ts
+
+### Remote & automatic
```

Comment:
I would mark it as a preferred way `### Remote & automatic (preferred)`

Reply:
(it's not working yet)